build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.2'
+        classpath 'com.android.tools.build:gradle:3.2.1'
```

Comment:
Upgrading breaks the unit tests as. For some reason the shared resources required by the unit tests are not found when using the newer gradle version. So unless there is a really good reason to upgrade, I'd postpone the upgrade and wait for a newer version that hopefully also supports shared resources.